src/models/simScoreCache.js

```diff
@@ -0,0 +1,46 @@
+const {
+  Model,
+} = require('sequelize');
+
+/**
+   * RolesTopic table. Junction table between Roles and Topics to support many to many relationship.
+   *
+   * @param {} sequelize
+   * @param {*} DataTypes
+   */
+export default (sequelize, DataTypes) => {
+  class SimScoreCache extends Model {
+    static associate(models) {
+      SimScoreCache.belongsTo(models.Goal, { foreignKey: 'id', as: 'goalOne' });
+      SimScoreCache.belongsTo(models.Goal, { foreignKey: 'id', as: 'goalTwo' });
```

Comment:
Please add the reversed associations. You have SimScoreCache -> Goals. But not Goals -> SimScoreCache

Reply:
050b0985b - not sure how I feel about the naming choice for the `as`, but I'm also not sure it matters much. 🤷